test/lib/commands/publish.js

```diff
@@ -876,49 +876,61 @@ t.test('latest dist tag', (t) => {
     // this needs more than one item in it to cover the sort logic
     { version: '50.0.0' },
     { version: '100.0.0' },
+    { version: '101.0.0', deprecated: 'oops' },
     { version: '105.0.0-pre' },
   ]
 
-  t.test('PREVENTS publish when latest version is HIGHER than publishing version', async t => {
+  await t.test('PREVENTS publish when highest version is HIGHER than publishing version', async t => {
```

Comment:
`await` is needed for nested async tests I take it? Did a linter catch this?

Reply:
I don't think it's absolutely required, this is the "least likely to surprise us" way of doing things.  tap itself accounts for this, but it shouldn't have to.